cohorts/views/views.py

```diff
@@ -128,13 +128,21 @@ def cohorts_list(request, is_public=False):
             'value': int(cohort['id']),
             'label': escape(cohort['name'])
         })
-        
+
+    cohort_data_versions = []
+    for cohort in cohorts:
+        cohort_data_versions.append({
+            'id': cohort.id,
+            'version': str(cohort.get_data_versions()[0])
```

Comment:
This will only grab the first version, and there might be a few. The best thing to do here is to add a special get_displays() for the whole query set. Go ahead and adjust this to:

`"; ".join(cohort.get_data_versions().get_displays())` 

And I'll add in the helper method. It'll produce an array of strings you can then join

Reply:
I have made the change and confirmed it works. I accidentally checked into master. So this PR no longer have any difference. Should it be closed?